server/service/src/invoice/prescription/update/update_picked_date.rs

```diff
@@ -0,0 +1,29 @@
+use chrono::Utc;
+use repository::{
+    InvoiceRow, InvoiceRowRepository, InvoiceStatus, InvoiceType, RepositoryError,
+    StorageConnection,
+};
+
+pub fn update_picked_date(
+    connection: &StorageConnection,
+    invoice: &InvoiceRow,
+) -> Result<(), RepositoryError> {
+    // We only want to update the picked date if the invoice is in the picked status and is a prescription
+    if invoice.status != InvoiceStatus::Picked {
+        return Ok(());
+    }
+
+    if invoice.r#type != InvoiceType::Prescription {
```

Comment:
Because of this condition, I think this function can live in the prescription update section, even though it's not called from here...